src/acq/adc.c

```diff
@@ -462,7 +462,8 @@ enum bl_error bl_acq_adc_configure(bl_acq_adc_t *adc,
 	}
 
 	/* ADC Group must have been initialized/configured first. */
-	uint32_t smp_max = adc->group->config.frequency / frequency;
+	uint32_t smp_max = adc->group->config.frequency /
+									(frequency * bl_led_count);
```

Comment:
Please do this arithmetic with frequency_trigger rather than in the code here. It's initialized in bl_acq_start.

Reply:
+1